src/components/MonthCalendar.spec.tsx

```diff
@@ -169,4 +169,52 @@ describe('MonthCalendar', () => {
             });
         });
     });
+
+    it('shows back and forward widget buttons', () => {
+        mountWithProviders(
+            <MonthCalendar events={ dummyEvents } focusDate={ new Date(2021, 4, 10) } onFocusDate={ () => null }  />,
+        );
+        cy.get('[data-testid="back-button"]').should('be.visible');
+        cy.get('[data-testid="fwd-button"]').should('be.visible');
+    });
+
+    it('shows the current displayed month in the widget', () => {
+        mountWithProviders(
+            <MonthCalendar events={ dummyEvents } focusDate={ new Date(2021, 4, 10) } onFocusDate={ () => null }  />,
+        );
+        cy.get('[data-testid="selected-month"]').contains('May');
+        cy.get('[data-testid="selected-month"]').contains('2021');
+    });
+
+    it('sets the focus date a month ago when back is clicked', () => {
+        const spyOnFocusDate = cy.spy();
+        mountWithProviders(
+            <MonthCalendar events={ dummyEvents } focusDate={ new Date(2021, 4, 10) } onFocusDate={ spyOnFocusDate }  />,
+        );
+
+        cy.findByText('misc.calendar.prev')
+            .click({ force: true })
+            .then(() => {
+                const date = new Date(2021, 3, 10);
+                expect(spyOnFocusDate).to.be.calledOnce;
+                expect(spyOnFocusDate.args[0][0] instanceof Date).to.be.true;
+                expect(spyOnFocusDate.args[0][0].toString()).to.eq(date.toString());
+            });
+    });
+
+    it('sets the focus date a month forward when next is clicked', () => {
+        const spyOnFocusDate = cy.spy();
+        mountWithProviders(
+            <MonthCalendar events={ dummyEvents } focusDate={ new Date(2021, 4, 10) } onFocusDate={ spyOnFocusDate }  />,
+        );
+
+        cy.findByText('misc.calendar.next')
+            .click({ force: true })
+            .then(() => {
+                const date = new Date(2021, 5, 9);
```

Comment:
I feel it's surprising that the date is not 2021-06-10, but instead 2021-06-09. I guess that's because of the shift by 30 days, but would it be very difficult to instead shift by one month?